### PR TITLE
[Sounds] Add audioplayer to SOUNDS library and made new library STEREOSOUNDS

### PR DESCRIPTION
**Pull request by MontyPython28**
# Add audioplayer to SOUNDS library and made new library STEREOSOUNDS

- Have transferred the SOUNDS library to modules,
- Have made new library called STEREOSOUNDS which produces stereo sounds. 
- Both Sound and StereoSound tabs, when spawned, contain an audio player that allows users to play their sound, pause their sound, adjust the volume of the sound and download the sound.

Fixes # (issue)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code quality imporvements

# How Has This Been Tested?

- Navigate to an instance of cadet-frontend, while running modules
- To test STEREOSOUNDS, run the following code:
```
import { make_stereo_sound, sine_sound, play, pan_mod } from "stereo_sounds";
const test = make_stereo_sound(t => math_sin(2 * 440 * math_PI * t), t => 0, 5);
const test2 = sine_sound(0.3, 70);
play(pan_mod(test2)(test));
```
- To test SOUNDS, run the following code: 
```
import { consecutively, triangle_sound, piano, violin, sine_sound, play, simultaneously } from "sounds";
const n1 = consecutively(list(triangle_sound(500,3), piano(66, 4)));
const n2 = consecutively(list(violin(60, 2), sine_sound(400, 5)));
play(simultaneously(list(n1, n2)));
```
- Both Sound tab and StereoSound tab should contain an audio player. Try playing, pausing, adjusting volume, downloading the sound as a file.
# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by MarcusTXK** [comment]
Hi, could you rebase onto the latest master branch? There are some conflicts that have to be resolved first before the PR can be made.

File: src/bundles/sounds/riffwave.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
+/* eslint-disable */
```
Comment (MarcusTXK) [inline src/bundles/sounds/riffwave.ts:1]: Similar to the previous one, please avoid disabling eslint and prettier totally. Try to conform to the rules set if possible, if not add `// eslint-disable-next-line rule-to-disable` to disable the specific rule for the next line.

Ill avoid pointing this out for future files, but please do the same for the rest.

**Review by MarcusTXK — COMMENTED** [review]
Hi, I left some suggestions for the code, please take a look at them and feel free to discuss if there is anything you disagree with.

**Comment by MontyPython28** [comment]
Hi Marcus! Finished rebasing and editing most of the es-lint disable commands.
Haven't changed the list.ts files, mainly because they're copied directly from list.js. Those haven't been integrated into the modules yet, but they're used to implement some sound functions, so we have not modified them much.
Now, it should be able to merge and the code should work.

**Comment by MarcusTXK** [comment]
Alright looks good, tested the module and works well, great job! Alright then leave the list.ts for now, it can be updated when it is integrated into modules.

**Pull request merged**